.travis.yml

```diff
@@ -13,6 +13,7 @@ sudo: false
 
 env:
   - TEST_TARGET=default
+  - TEST_TARGET=default TEST_MINIMAL=true
   - TEST_TARGET=coding
```

Comment:
Maybe we can make as many of these "minimal" too - it might speed up some of our builds a little.